hadoop-ozone/common/pom.xml

```diff
@@ -48,6 +48,17 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
     <dependency>
       <groupId>io.netty</groupId>
       <artifactId>netty-handler-proxy</artifactId>
+    </dependency>
+      <dependency>
+        <groupId>io.netty</groupId>
+        <artifactId>netty-tcnative-boringssl-static</artifactId>
+        <version>2.0.38.Final</version> <!-- See table for correct version -->
```

Comment:
Netty provides a BOM (bill of materials) module.  We use it to ensure version consistency of various Netty modules (see HDDS-5298).  In most recent versions this BOM also includes tcnative modules.  We will be able to avoid tracking tcnative version after upgrade to latest Netty (HDDS-6097).

Reply:
Thanks @hanishakoneru for reviewing this PR and for your comments.  I've added the pom property for the netty-tcnative version to the central main pom.xml.  Thanks.
`<tcnative.version>2.0.38.Final</tcnative.version>`

I also brought this up with @adoroszlai offline and he kindly added a comment to this PR on this.